src/main/java/uk/gov/hmcts/divorce/divorcecase/NoFaultDivorce.java

```diff
@@ -20,7 +20,7 @@
 @Component
 public class NoFaultDivorce implements CCDConfig<CaseData, State, UserRole> {
 
-    public static final String CASE_TYPE = "NO_FAULT_DIVORCE18";
+    public static final String CASE_TYPE = "NFD";
```

Comment:
should be NO_FAULT_DIVORCE 🤔

Reply:
I thought we were changing it now to NFD? - https://github.com/hmcts/cnp-flux-config/pull/10235/files

Reply:
yes it's `NFD`